src/helpers/apiCalls.tsx

```diff
@@ -0,0 +1,22 @@
+export const getCoordinates = async (query: string) => {
+	const response = await fetch(`http://api.positionstack.com/v1/forward?access_key=e17943cbd88c595c58c3c6ae1840fc33&query=${query}`)
```

Comment:
Great Job! I know its not fully there but this is a big chunk of what we need to get done. Thank you!